components/alertbanner/metadata/alertbanner.yml

```
@@ -0,0 +1,116 @@
+name: Alert Banner
```

```suggestion
name: Alert banner
```

Minor nit - technically we're supposed to be using sentence case for headings rather than title case.